docs/src/single-pool.mdx

```diff
@@ -0,0 +1,477 @@
+```
+
+### Keypair
+Web3.js Next uses [`crypto.subtle`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle) for key management. For local development, one can create an object `{ privateKey, publicKey }` using `crypto.subtle.importKey` to produce signing (private) and verifying (public) keys. Generally, however, one should allow the user wallet to manage keys, as the system when used properly is designed to not allow key material to leave secure storage.
+
+  </TabItem>
+</Tabs>
+
+### Setting up a single-validator pool
+
+#### Creating the pool
+
+A single-validator stake pool can be created permissionlessly, by anyone, for a
+given vote account. This allows you to receive the full staking yield you would by
+staking directly while holding the value in a tokenized form.
+It also allows you to buy or sell stakes smaller than the minimum delegation on the market.
+
+Assuming a vote account `Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC` exists, we create a pool at address `DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb`:
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool initialize Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC
+```
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const voteAccountAddress = new PublicKey('Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC');
+const transaction = await SinglePoolProgram.initialize(
+  connection,
+  voteAccountAddress,
+  feePayerAddress,
+);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+```typescript
+const voteAccountAddress = 'Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC' as VoteAccountAddress;
+const transaction = await SinglePoolProgram.initialize(
+  rpc,
+  voteAccountAddress,
+  feePayerAddress,
+);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+</Tabs>
+
+#### Managing token metadata
+
+By default, when a pool is created, it also creates Metaplex token metadata for the mint associated with the pool.
+If, for whatever reason, this was opted out of by the pool creator, anyone may create the default metadata permissionlessly:
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool create-token-metadata --pool DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb
+```
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const transaction = await SinglePoolProgram.createTokenMetadata(poolAddress, feePayerAddress);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+```typescript
+const transaction = await SinglePoolProgram.createTokenMetadata(poolAddress, feePayerAddress);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+</Tabs>
+
+The default token metadata is only minimally helpful, spotlighting the address of the pool.
```

Comment:
is this correct?
```suggestion
The default token metadata is only minimally helpful, spotlighting the address of the validator vote account.
```

Reply:
oops youre right! i was looking at the create with seed call when i wrote this..